wizard/WizardDaemonSettings.qml

```diff
@@ -94,6 +94,13 @@ ColumnLayout {
             }
         }
 
+        MoneroComponents.CheckBox {
+            id: pruneBlockchainCheckBox
+            checked: persistentSettings.pruneBlockchain
+            onClicked: persistentSettings.pruneBlockchain = true
```

Comment:
Is it possible to uncheck it again? Seems like this checkbox can only be set to true.

Reply:
Oops, thanks